contributors/devel/release/issues.md

```diff
@@ -70,92 +70,111 @@ This will also be used to escalate to the correct SIG GitHub team.
 - `kind/feature`: New functionality.
 - `kind/cleanup`: Adding tests, refactoring, fixing old bugs.
 
+## Bot communication
+
+The bot will communicate the state of an issue in the active milestone via comments and labels.
+
+### Comments
+
+All bot comments will mention issue owners and link to this doc.  Bot
+comments in the workflow section of this document will only include
+the message but be presumed to include the header and footer in the
+following example:
+
+```
+@pwittrock @droot
+
+<message>
+
+Additional instructions available [here](<link to this doc>)
+```
+
+### Labels
+
+The following labels are used by the bot to track the state of an
+issue in the milestone:
+
+ - status/incomplete-labels - one or more required labels is missing
+ - status/needs-approval - the `status/approved-for-milestone` label is missing
+ - status/needs-attention - a status label is missing or an update is required
+ - status/milestone-removed - the issue was removed from the milestone
+
+These labels are mutually exclusive - only one will appear on an issue at once.
```

Comment:
Seems like `status/incomplete-labels` and `status/needs-approval` aren't mutually exclusive, i.e., "when the `status/approved-for-milestone` label is missing" is one of the cases on "one or more required labels is missing".

Reply:
`status/incomplete-labels` is applied when the `kind/` `priority/` and `sig/` labels are not correct.  Only once those labels are correct (and `status/incomplete-labels` is removed) will `status/approved-for-milestone` be checked for and `status/needs-approval` be applied.

Reply:
I've updated the explanation for `status/incomplete-labels`.  Does that help?

Reply:
>Only once those labels are correct (and status/incomplete-labels is removed) will status/approved-for-milestone be checked for and status/needs-approval be applied if it is missing.

Got it. Do we need to also explain this somewhere in the document? Otherwise readers may still be confused why  `status/incomplete-labels` and `status/needs-approval` are mutually exclusive.

>I've updated the explanation for status/incomplete-labels

It's more clear now. One more question, should we take `committee/*` labels into consideration? Some issues may not need sig labels but committee labels.

Thanks!

Reply:
When would `committee/*` labels be needed?  Issues targeted by this document will only be actionable (be able to be implemented by code) and targeted at a milestone.

Reply:
I only find some issues with `committee/conduct` [here](https://github.com/kubernetes/community/issues?q=is%3Aissue+is%3Aopen+label%3Acommittee%2Fsteering) but seems like they don't target at any milestone and not actionable. So we can ignore it in this doc, IMO.